core/chains/evm/gas/models.go

```diff
@@ -290,13 +301,21 @@ func (e *WrappedEvmEstimator) BumpFee(ctx context.Context, originalFee EvmFee, f
 				TipCap: originalFee.DynamicTipCap,
 				FeeCap: originalFee.DynamicFeeCap,
 			}, feeLimit, maxFeePrice, attempts)
+		if err != nil {
+			return
+		}
+		chainSpecificFeeLimit, err = commonfee.ApplyMultiplier(chainSpecificFeeLimit, e.geCfg.LimitMultiplier())
 		bumpedFee.DynamicFeeCap = bumpedDynamic.FeeCap
 		bumpedFee.DynamicTipCap = bumpedDynamic.TipCap
 		return
 	}
 
 	// bump legacy fee
 	bumpedFee.Legacy, chainSpecificFeeLimit, err = e.EvmEstimator.BumpLegacyGas(ctx, originalFee.Legacy, feeLimit, maxFeePrice, attempts)
+	if err != nil {
+		return
+	}
+	chainSpecificFeeLimit, err = commonfee.ApplyMultiplier(chainSpecificFeeLimit, e.geCfg.LimitMultiplier())
 	return
 }
 
```

Comment:
I can't place the comment any lower but looks like there are a couple of methods below that are used by the Block History Estimator bump methods that are also applying the multiplier. `BumpLegacyGasPriceOnly` and `BumpDynamicFeeOnly`.